docker/dev/docker-compose.yml

```diff
@@ -77,9 +77,8 @@ services:
       NODE_ENV: development
       PORT: 3004
       DATABASE_URL: postgres://boutique_backend:boutique_backend@postgres/boutique_backend
-      PRIVATE_KEY: ${PRIVATE_KEY}
-      KEY_ID: ${KEY_ID}
-      PAYMENT_POINTER: ${PAYMENT_POINTER}
+      PRIVATE_KEY: LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1DNENBUUF3QlFZREsyVndCQ0lFSUI0YzgyOVlSMDZCTUhtQmpIVTNrOHZmMEZhVEFvOHNCTzIrRFZwY1lPSk0KLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLQ==
```

Comment:
IMO the default values should be in `.env.example` and not in docker-compose. That will allow devs to update them as needed without affecting the code

Reply:
I agree.

Reply:
Replying to @dragosp1011 and @beniaminmunteanu: I agree, the values should be in `.env.example`.

As a side note: I think that seeding process for the initial setup might be better handled as a script. This script could either run automatically at startup or be executed manually, rather than being integrated directly into the backend.